compose/auth_proxy.yml

```diff
@@ -5,12 +5,14 @@ services:
       - "${CRYOSTAT_HTTP_PORT}"
     environment:
       CRYOSTAT_HTTP_PROXY_HOST: auth
-      CRYOSTAT_HTTP_PROXY_PORT: '8080'
-      QUARKUS_HTTP_PROXY_PROXY_ADDRESS_FORWARDING: 'true'
-      QUARKUS_HTTP_PROXY_ALLOW_X_FORWARDED: 'true'
-      QUARKUS_HTTP_PROXY_ENABLE_FORWARDED_HOST: 'true'
-      QUARKUS_HTTP_PROXY_ENABLE_FORWARDED_PREFIX: 'true'
-      QUARKUS_HTTP_PROXY_TRUSTED_PROXIES: 127.0.0.1:${CRYOSTAT_HTTP_PORT}
+      CRYOSTAT_HTTP_PROXY_PORT: '8443'
```

Comment:
I think we might have talked about this before, but the proxy can be configured with `--http-address`, `--https-address`, and `--force-https` (these should all also be convertable to env. vars.) so that it could continue listening on HTTP 8080 but redirect to HTTPS 8443.

Or using the Alpha options config: https://oauth2-proxy.github.io/oauth2-proxy/configuration/alpha-config/#server